src/routers/alpha-router/quoters/v2-quoter.ts

```diff
@@ -213,6 +218,20 @@ export class V2Quoter extends BaseQuoter<V2CandidatePools, V2Route, Token> {
       MetricLoggerUnit.Count
     );
 
+    metric.putMetric(
+      `V2QuotesFetched_Chain_${this.chainId}`,
+      _(routesWithQuotes)
+        .map(([, quotes]) => quotes.length)
+        .sum()
+    );
+
+    metric.putMetric(`V2RoutesFetched`, _(routesWithQuotes).sum());
```

Comment:
so this logs the routes fetched in case we have a cache routes miss. in case we have a cached routes hit, it will be within [refreshRoutesThenGetQuotes](https://github.com/Uniswap/smart-order-router/pull/756/files#diff-e2eb9d7f54a7e353f88a2a325fb275b4a08e31c49661fa3851bc87429e30d2a6R302).

I'm thinking (1) whether we should log routes and/or quotes fetched in case of cached routes hit (2) whether we should log both cached routes hit and cached routes miss

Reply:
makes sense - lets chat